<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qapplication::LogWindow</class>
 <widget class="QMainWindow" name="qapplication::LogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="log_view"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="window_menu">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="window_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="search_dock">
   <property name="windowTitle">
    <string>Search</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="search_dock_content">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="filter_edit">
         <property name="placeholderText">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_filter_button">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset theme="edit-clear"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="filter_column_label">
         <property name="text">
          <string>Column: </string>
         </property>
         <property name="buddy">
          <cstring>filter_column</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filter_column"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filter mode: </string>
         </property>
         <property name="buddy">
          <cstring>filter_mode</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filter_mode"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="filter_case_sensitive">
         <property name="text">
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="message_type_toolbar">
   <property name="windowTitle">
    <string>Message types</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="show_fatal_action"/>
   <addaction name="show_critical_action"/>
   <addaction name="show_warning_action"/>
   <addaction name="show_info_action"/>
   <addaction name="show_debug_action"/>
  </widget>
  <action name="show_fatal_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fatal</string>
   </property>
   <property name="toolTip">
    <string>Show fatal message</string>
   </property>
  </action>
  <action name="show_critical_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Critical</string>
   </property>
   <property name="toolTip">
    <string>Show critical messages</string>
   </property>
  </action>
  <action name="show_warning_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
   <property name="toolTip">
    <string>Show warning messages</string>
   </property>
  </action>
  <action name="show_info_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Show information messages</string>
   </property>
  </action>
  <action name="show_debug_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Show debug messages</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear_filter_button</sender>
   <signal>clicked()</signal>
   <receiver>filter_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
